mtp_api/apps/mtp_auth/tests/test_views.py

```diff
@@ -52,6 +53,61 @@ def prison_set(user):
     )
 
 
+class JobInformationTestCase(APITestCase, AuthTestCaseMixin):
+    fixtures = [
+        'initial_groups.json',
+        'initial_types.json',
+        'test_prisons.json',
+    ]
+
+    url = reverse('job-information-list')
+
+    payload = {
+        'title': 'Warden',
+        'prison_estate': 'Private',
+        'tasks': 'Run the show',
+    }
+
+    def setUp(self):
+        super().setUp()
+        test_users = make_test_users()
+        self.security_staff = test_users['security_staff'][0]
+        self.security_staff.current_password = 'foobar'
```

Comment:
what is this line for? all the test users have their username as their password

Reply:
Ah, I didn't know that - but this was also legacy code from a previous method of auth anyway - by using the post request with `HTTP_AUTHORIZATION` we don't need it anyway.